rmap-loader-deposit-disco/src/main/java/info/rmapproject/loader/deposit/disco/DiscoDepositConsumer.java

```diff
@@ -139,8 +138,7 @@ public void accept(HarvestRecord record) {
             if (response.getStatusLine().getStatusCode() == 201) {
                 harvestRegistry.register(
                         record.getRecordInfo(),
-                        URI.create(response.getFirstHeader("Location")
```

Comment:
What's the motivation for this change?  Just by knowing HTTP/1.1, the client can presume that the Location header has a URI to the created resource; which is sufficient for the purposes of the harvester.  Is there a particular advantage to using the URI from the entity body?